### ISSUE TITLE
Hide some infos for livestreams

### ISSUE THREAD

**Issue by Razzeee** [phil65/skin.estuary#247]
Livestreams have some confusing ui elements:
- Progressbar
- Current timestamp
- Total timestamp

We can access if it's a stream via `Player.IsInternetStream`.
Not sure it's wise to do that as it might be a valid usecase for "not-live-streams-but-streams"?

@ronie @phil65 @HitcherUK

**Comment by ronie** [comment]
these 'confusing' timestamps are provided by core.
so it should be fixed there imo.

**Comment by Razzeee** [comment]
Agreed, implementing an interface like pvr uses might be the best thing to do. I think @peak3d mentiond that that's the plan for inputstream.
